driver/CMakeLists.txt

```diff
@@ -28,7 +28,12 @@ cmake_minimum_required( VERSION 3.5)
 find_package(Threads REQUIRED)
 
 add_executable(MIOpenDriver main.cpp InputFlags.cpp)
+target_include_directories(MIOpenDriver PRIVATE ${MLIRMIOpen_INCLUDE_DIR})
```

Comment:
If the name of the package is `miir` then why are the variable names following the old convention?

Reply:
I was thinking `MLIRMIOpen` is the library name. 
And our team is in the middle of name changing. The `Miir` can also be changed in the near future.

Reply:
That's right. We plan rename all those in ROCm 5.4 time frame (including the MLIR project name!). It's going to me `rocMLIR` and `rock` dialect. Project names referenced here will change.

Speaking of which @zhanglx13 maybe we want to talk to @sjw36  and determine the name so the package name can go in as the ultimate form. We can amend `miir` header name in another PR?

Reply:
@jerryyin Sure. Changing package and library names can be done any time and it should not affect developer on the MIOpen team as long as wedo not change the API name. 
Currently I am figuring out the test failures, which has something to do with the dependencies. After everything is resolved, this PR should be good to go.

Reply:
Sounds good to me. Just a heads up though, the name change will have to happen inside a single release otherwise we'll be facing with backward compatibility breakages. Let's coordinate it after this PR.

Reply:
This should not be added here. Instead you should do `target_internal_library(MIOpen Miir::MLIRMIOpen)` so the usage requirements of mlir are propagated to the driver and the tests.

Reply:
removed since MIOpenDriver is not including Miir.h